gcc/rust/rust-session-manager.cc

```diff
@@ -574,13 +574,16 @@ Session::enable_dump (std::string arg)
 /* Actual main entry point for front-end. Called from langhook to parse files.
  */
 void
-Session::parse_files (int num_files, const char **files)
+Session::handle_input_files (int num_files, const char **files)
 {
+  if (num_files != 1)
+    rust_fatal_error (Location (),
+		      "only one file may be specified on the command line");
```

Comment:
I am not sure if we want to match `rustc`'s error message closely.

`rustc` will emit this message when multiple files are specified:

```
error: multiple input filenames provided (first two filenames are `main.rs` and `lib.rs`)
```

Reply:
I think rustc's message is great :DD We can improve this one of course